common/src/main/java/org/wso2/testgrid/common/config/InfrastructureConfig.java

```diff
@@ -43,6 +43,8 @@
     private ContainerOrchestrationEngine containerOrchestrationEngine;
     private Properties parameters = new Properties();
     private List<Provisioner> provisioners;
+    private List<String> excludes;
```

Comment:
IMO, something like "excludeInfraParams" might add more meaning since we are only excluding infra parameters. :)  WDYT?

Reply:
Since Include and exclude attributes are defined under the infrastructureConfig: section it indicates that include and exclude attributes are related to infrastructure. Therefore lest's keep attribute names as this way. 

Further with this improvement infrastructure_parameter table has to be modified. That means we should modify infrastructure parameter names as following way.

>mysql ---->  mysql-5.7
>postgres ----> postgres-10.3
>centos ----> centos-7.4